common/src/main/java/com/wynntils/models/territories/TerritoryModel.java

```diff
@@ -173,12 +166,12 @@ private void updateTerritoryProfileMap() {
                     allTerritoryPois = territoryProfileMap.values().stream()
                             .map(TerritoryPoi::new)
                             .collect(Collectors.toSet());
-                    // TODO: Add events if territories changed
                 },
                 onError -> {
                     errorCount++;
                     if (errorCount >= MAX_ERRORS) {
-                        WynntilsMod.error("Athena user lookup has repeating failures. Disabling future lookups.");
+                        WynntilsMod.error(
```

Comment:
I think we can remove this entire errorCount thing. It was just a workaround to handle bad Athena behavior. Such a failure is much more unlikely with Wynncraft.

Reply:
Perhaps.. I just kept it as a fallback.